### PR TITLE
Update SmartThings hub and outlet to the new resource model.

### PR DESCRIPTION
**Pull request by m-chau**

### CODE REVIEW

**Comment by arjun-msft** [comment]
:shipit:

**Comment by arjun-msft** [comment]
This also looks good to me also, can someone else have a quick peek?

File: opent2t.p.outlet/com.smartthings.binaryswitch/js/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "test": "ava"
+  },
+  "ava": {
+    "files": [
+      "tests/*.js"
+    ]
+  },
+  "dependencies": {
+    "opent2t": "*",
+    "opent2t-translator-com-smartthings-hub": "*"
```
Comment (ChuckFerring) [inline opent2t.p.outlet/com.smartthings.binaryswitch/js/package.json]: >"*" [](start = 46, length = 3)

Should this be a specific or minimum version.

File: opent2t.p.outlet/com.smartthings.binaryswitch/js/thingTranslator.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        deviceId = deviceInfo.deviceInfo.opent2t.controlId;
+        smartThingsHub = deviceInfo.hub;
+
+        console.log('SmartThings Binary Switch initializing...Done');
+    }
+
+    // exports for the entire schema object
+
+    // Queries the entire state of the lamp
+    // and returns an object that maps to the json schema org.opent2t.sample.lamp.superpopular
```
Comment (ChuckFerring) [inline opent2t.p.outlet/com.smartthings.binaryswitch/js/thingTranslator.js]: Nit: the comments look like they were copied from a lamp translator, please update.

**Comment by ChuckFerring** [comment]
Had a couple minor comments but no blockers.

**Comment by begillil** [comment]
:shipit:

**Pull request merged**